azurerm/resource_arm_key_vault_secret_test.go

```diff
@@ -153,6 +153,9 @@ func TestAccAzureRMKeyVaultSecret_complete(t *testing.T) {
 				Config: config,
 				Check: resource.ComposeTestCheckFunc(
 					testCheckAzureRMKeyVaultSecretExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "not_before_date", "2019-01-01T01:02:03Z"),
+					resource.TestCheckResourceAttr(resourceName, "expiration_date", "2020-01-01T01:02:03Z"),
+					resource.TestCheckResourceAttr(resourceName, "tags.hello", "world"),
```

Comment:
The same check is done just a couple lines below. Can we remove this one?

Reply:
whoops, removed!